README.md

```diff
@@ -46,26 +48,338 @@ ember install ember-resources
 ## Example
 
 ```js
-import { useFunction } from 'ember-resources';
+import { trackedFunction } from 'ember-resources';
 
 class MyClass {
-  data = useFunction(this, async () => {
-    let response = await fetch('...');
+  @tracked endpoint = 'starships';
+
+  data = trackedFunction(this, async () => {
+    let response = await fetch(`https://swapi.dev/api/${this.endpoint}`);
     let json = await response.json();
-    return json;
+
+    return json.results;
   }),
+
+  get records() {
+    return this.data.value ?? [];
+  }
+}
+```
+```hbs
+{{this.records}}
+```
+
+In this example, `trackedFunction` will make a call to [StarWars API](https://swapi.dev/)
```

Comment:
_Maybe_ it will be better than putting it into words, paste a code there? Or maybe both? 

```js

@action
switchToPlanets() {
  this.endpoint = 'planets';
}
```